.github/workflows/redis.yml

```diff
@@ -404,9 +410,7 @@ jobs:
        poetry install --no-root
    - name: build redis from source
      if: steps.redis-already-built.outcome != 'success'
-     run: |
-       cd redis
-       make all BUILD_TLS=yes FINAL_LIBS="-lm -ldl ../deps/hiredis/libhiredis_ssl.a /usr/local/opt/openssl/lib/libssl.a /usr/local/opt/openssl/lib/libcrypto.a"
+     run: bash redis-builder/scripts/build-osx.sh
```

Comment:
I am honestly shocked this works. The whole this was an experiment of mine.

`
- name: build redis from source
  working-directory: redis-builder/
  run: |
   bash scripts/build-osx.sh
`

As the build step is [skipped](https://github.com/redis-stack/redis-stack/actions/runs/7637725534/job/20807170477?pr=501) since the binaries already exist, let's test this first.

Reply:
Ah, sorry, didn't notice this! Should I remove condition to "force" rebuild?